mkdocs.yml

```diff
@@ -1,3 +1,4 @@
 site_name: djinni-support-lib
 nav:
-  - "install.md"
\ No newline at end of file
+  - install.md
+  - developer-guide.md
```

Comment:
Important: If you add new pages to the documentation they have to be referenced here for them to appear in the generated website! This was forgotten for the developer guide